package.json

```diff
@@ -41,6 +41,6 @@
     "file-loader": "^0.8.5",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
-    "webpack": "^1.13.1"
+    "webpack": "^1.15.0"
```

Comment:
I dont think it is strictly required .

Reply:
Okay, can you roll it back? We shouldn't be unnecessarily updating dependencies.